config/routes.rb

```diff
@@ -1,11 +1,12 @@
 Rails.application.routes.draw do
 
-  root 'programs#show', id: 1
+  root 'nodes#show', id: nil
```

Comment:
I may be wrong, but I think you want to link root with 'nodes#index' (so the path will be /nodes without any id) Or am I misunderstanding something?